From fd6a09dd731b80043227b74e77cdd130d9a49101 Mon Sep 17 00:00:00 2001
From: Paul Seidler <bl4sph3my@online.de>
Date: Mon, 7 Feb 2011 17:37:51 +0000
Subject: [PATCH] test: use localhost

---
 tests/twisted/idletest.py |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/twisted/idletest.py b/tests/twisted/idletest.py
index 7a586ad..e1e0fe8 100644
--- a/tests/twisted/idletest.py
+++ b/tests/twisted/idletest.py
@@ -40,7 +40,7 @@ class BaseIRCServer(irc.IRC):
 
     def listen(self, port, factory):
         self.log ("BaseIRCServer listening...")
-        return reactor.listenTCP(port, factory)
+        return reactor.listenTCP(port, factory, interface='localhost')
 
     def connectionMade(self):
         self.log ("connection Made")
@@ -110,7 +110,7 @@ class SSLIRCServer(BaseIRCServer):
         key_file = os.environ.get('IDLE_SSL_KEY', 'tools/idletest.key')
         cert_file = os.environ.get('IDLE_SSL_CERT', 'tools/idletest.cert')
         return reactor.listenSSL(port, factory,
-                ssl.DefaultOpenSSLContextFactory(key_file, cert_file))
+                ssl.DefaultOpenSSLContextFactory(key_file, cert_file), interface='localhost')
 
 def install_colourer():
     def red(s):
-- 
1.7.4

